### ISSUE TITLE
Stop doing 14/20 key checks on first use of app

### ISSUE THREAD

**Issue by Stephen-PM** [cds-snc/covid-shield-mobile#642]
We are currently seeing either 14 or 20 checks in the setting->checks when a user installs the app and uses it for the first time. My understanding is that this is only useful if the user had previously had the app running on the phone and collected keys from encounters, uninstalled the app, then re-installed the app. Are there other reasons why we need this. @timarney @maxneuvians @obrien-j 

This is impacting our ability to run experiments because the app is hitting the 20 checks per day limit and is then unable to perform checks in the background until the next day.

One solution could be to avoid pulling down keys for dates that are earlier than the first keys logged on the phone.

**Comment by henrytao-me** [comment]
### Action items: 

- [ ] Create new endpoint to fetch last 14 days (exclude today keys). Consume new endpoint on mobile which should be run only once after onboarding flow. 
- [ ] Investigate why mobile makes 20 exposure checks instead of 14 after boarding flow. It could be related to this i18n issue https://github.com/cds-snc/covid-shield-mobile/issues/583
- [ ] Android only: add option in Demo Mode to switch between "deferred" and "exact" mode for background job. Default is "deferred". See https://developer.android.com/guide/background

**Comment by timarney** [comment]
On an allow listed Android device I'm seeing 28 exposure checks which would seem to map back to #583

**Comment by henrytao-me** [comment]
Close as the last PR will be handled by this issue https://github.com/cds-snc/covid-shield-mobile/issues/583
